Formula/d/dependabot.rb

```diff
@@ -0,0 +1,26 @@
+class Dependabot < Formula
+  desc "Tool for testing and debugging Dependabot update jobs"
+  homepage "https://github.com/dependabot/cli"
+  url "https://github.com/dependabot/cli/archive/refs/tags/v1.54.0.tar.gz"
+  sha256 "ae496c65e4f947cca1251c9f786fbcc1e5e1a079af7fee08b893265e69d39295"
+  license "MIT"
+  head "https://github.com/dependabot/cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/dependabot/cli/cmd/dependabot/internal/cmd.version=#{version}
+    ]
+    system "go", "build", *std_go_args(output: bin/"dependabot", ldflags:), "./cmd/dependabot"
```

Comment:
since formula name and artifact name are the same, you dont need output anymore.

```suggestion
    system "go", "build", *std_go_args(ldflags:), "./cmd/dependabot"
```